shared/src/main/scala/io/lemonlabs/uri/parsing/UrlParser.scala

```diff
@@ -257,14 +257,14 @@ class UrlParser(val input: ParserInput)(implicit conf: UriConfig = UriConfig.def
   val extractFullIpv6 = (pieces: immutable.Seq[String]) => IpV6.fromHexPieces(pieces)
 
   val extractIpv6WithEluded = (beforeEluded: immutable.Seq[String], afterEluded: immutable.Seq[String]) => {
```

Comment:
`Eluded` is a typo here and should be `Elided`. Can't change it now because it breaks binary compatibility as mima. I will fix at the next big version bump and also make these fields `private` as they need not be public.